test/vagrant-service-manager/binary_handlers/adb_docker_binary_handler_test.rb

```diff
@@ -54,6 +54,11 @@ module ServiceManager
       end
 
       it 'should validate download url' do
+        @handler.build_download_url
+        assert_silent { @handler.validate_url }
```

Comment:
This is an odd test. You are saying that just because there is no output, the validation succeeded. Why does validate_url not return something _true_ or something? Can you not test some actual URLs and also trigger the exceptional case. You might want to pass the url into the method instead of using @url. This way you can test this method much easier.